UnityProject/Assets/Scripts/Items/Weapons/Meleeable.cs

```diff
@@ -64,7 +64,7 @@ public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
 		}
 
 		//not punching unless harm intent
-		if (interaction.HandObject == null && interaction.Intent != Intent.Harm) return false;
+		if (interaction.Intent != Intent.Harm) return false;
```

Comment:
what is this change for?

Reply:
'remove the ability to melee people on intents other then harm'
as in the commit title